save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPluginConfig.kt

```diff
@@ -57,6 +61,9 @@ data class WarnPluginConfig(
     val testNameSuffix: String? = null,
 ) : PluginConfig {
     override val type = TestConfigSections.WARN
+
+    @Transient
+    override var configLocation: Path = "undefined_toml_location".toPath()
```

Comment:
please check how ktoml will handle it

Reply:
I add `configLocation` to toml file and received the following:
`Unknown key received: <configLocation> in scope ...`, so I think all is okay, and `@Transient` hides this variable from the
serialization process.

I also enabled `PerformAnalysisTest`, since after https://github.com/cqfn/save/pull/120 all worked fine.